wled00/bus_manager.h

```diff
@@ -296,7 +295,7 @@ class BusPwm : public Bus {
     if (_type == TYPE_ANALOG_3CH && _cct >= 1900) {
       c = colorBalanceFromKelvin(_cct, c); //color correction from CCT
     }
-    c = autoWhiteCalc(c);
+    if (_type != TYPE_ANALOG_3CH) c = autoWhiteCalc(c);
```

Comment:
Auto-White should be done prior to color correction, otherwise it gets off.

Reply:
Technically doesn't matter in this case since auto white is only done for busses other than 3CH RGB while correction is only done for 3CH RGB. I'll swap these around for better maintainability though, also I think correction should also be done for the 4CH RGBW type.